legal-api/src/legal_api/resources/v2/business/business.py

```diff
@@ -193,6 +193,18 @@ def search_businesses():
         return {'error': 'Unable to retrieve businesses.'}, HTTPStatus.INTERNAL_SERVER_ERROR
 
 
+@bp.route('/search/affiliation_mappings', methods=['GET'])
+@cross_origin(origin='*')
+@jwt.requires_roles([SYSTEM_ROLE])
+def get_filing_details():
+    """Return the list of Business filings with name requests. Being called from auth api."""
+    results = BusinessSearchService.get_affiliation_mapping_results()
```

Comment:
I don't think you want the entire database, you probably want some identifiers 

https://github.com/bcgov/sbc-auth/blob/94ab324560dd36bdd907c148c4638f1c214eb9e1/auth-api/src/auth_api/services/entity_mapping.py#L148